README.md

```diff
@@ -417,6 +417,7 @@ Variable | Description | Default value
 `JVB_PORT` | UDP port for media used by Jitsi Videobridge | 10000
 `JVB_TCP_HARVESTER_DISABLED` | Disable the additional harvester which allows video over TCP (rather than just UDP) | true
 `JVB_TCP_PORT` | TCP port for media used by Jitsi Videobridge when the TCP Harvester is enabled | 4443
+`JVB_COLIBRI_PORT` | TCP port for Colibri statistics provided by Jitsi Videobridge | 8080
```

Comment:
This change needs to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)